src/Illuminate/Foundation/PackageManifest.php

```diff
@@ -58,16 +58,27 @@ public function __construct(Filesystem $files, $basePath, $manifestPath)
         $this->vendorPath = $basePath.'/vendor';
     }
 
+    /**
+     * Get all of the $name values for all packages.
+     *
+     * @param  string  $name
+     * @return array
+     */
+    public function extra($name)
```

Comment:
I don't think this method name is very clear in explaining what it does?

Reply:
Open to suggestions if you have anything particular in mind.

Could be updated to `getExtra()` or `getExtraSegment()` if you see that as reasonable